Sources/ObjCSupport/LOKLabelLayout.swift

```diff
@@ -9,6 +9,15 @@
 import UIKit
 
 @objc open class LOKLabelLayout: LOKBaseLayout {
+    @objc public let attributedString: NSAttributedString?
+    @objc public let string: String?
+    @objc public let font: UIFont
+    @objc public let numberOfLines: Int
+    @objc public let alignment: LOKAlignment
```

Comment:
Looks like flexibility is missing here.

Also, please add similar properties on LOKSizeLayout, LOKTextViewLayout, LOKOverlayLayout, and LOKButtonLayout. I realize that you don't currently need them yourself, but it doesn't make sense for the library to have them only on some classes and not on other similar classes.

Reply:
\- `flexibility` and `viewReuseId` (computed) properties are present under `LOKBaseLayout`, hence we haven't added it in child classes.
\- As per your suggestion, added similar properties in `LOKSizeLayout`, `LOKTextViewLayout`, `LOKOverlayLayout` and `LOKButtonLayout` classes.

Reply:
Good point regarding `flexibility` and `viewReuseId`, thanks.